scripts/ghutils.py

```diff
@@ -436,19 +435,32 @@ def create_release(repo, version, bodyfile):
     # Store version in context for use in notifications
     ctx_obj.version = version
     try:
-        release = repo.create_release(
-            tag_name=version,
-            name=version,
-            body=release_notes,
-        )
-        logger.info("Release {} created.".format(version))
+        try:
```

Comment:
Making use of `else:` prevents the exception from unintentionally being caught from the wrong line of code and, at least in my opinion, is more readable.
```suggestion
        try:
            release = [r for r in repo.iter_releases() if r.name == version][0]
        except IndexError:
            release = repo.create_release(
                tag_name=version,
                name=version,
                body=release_notes,
            )
            logging.info("Release {} created.".format(version))
        else:
            if release.body != release_notes:
                raise ReleaseFailureException(
                    "Release failed. Github release {version} already exists"
                    " but its release notes don't match the release notes"
                    " currently being supplied."
                    .format(version=version)
                )
            logging.info("Release {} already exists, not creating."
                         .format(version))
```

Reply:
done